scripts/utils.js

```diff
@@ -0,0 +1,49 @@
+const { readdir, readFile, writeFile, mkdir } = require('node:fs/promises');
+const path = require('path');
+const pify = require('pify');
+const xml2js = require('xml2js');
+
+const builder = new xml2js.Builder({
+  rootName: 'svg',
+  xmldec: { version: '1.0', encoding: 'UTF-8' }
+});
+
+async function gatherIcons(iconPath) {
+  const files = await readdir(iconPath);
+  const svgFiles = files.filter(f => f.indexOf('.svg') !== -1);
```

Comment:
Maybe a little more modern here too?

```suggestion
  const svgFiles = files.filter(f => f.endsWith('.svg'));
```